index.html

```diff
@@ -161,58 +230,64 @@ <h2 class="titulo-secundario  margem">Formação</h2>
         <div class="traco-pequeno"></div>
         <div class="traco-minusculo"></div>
       </div>
-      <div class="unesp-etec"> 
-        <div class="titulo-unifesp"> 
-          <img src="assets/unifesp-branco.png" alt="Logo da Universidade Federal de São Paulo" title="Universidade Federal de São Paulo"/>
-          <h3  class="titulo-terciario">Bacharelado em Ciência e Tecnologia - <b>UNIFESP</b></h3>
+      <div class="unesp-etec">
+        <div class="titulo-unifesp">
+          <img src="assets/unifesp-branco.png" alt="Logo da Universidade Federal de São Paulo"
+            title="Universidade Federal de São Paulo" />
+          <h3 class="titulo-terciario">Bacharelado em Ciência e Tecnologia - <b>UNIFESP</b></h3>
```

Comment:
```suggestion
          <h3 class="titulo-terciario">
              Bacharelado em Ciência e Tecnologia - <b>UNIFESP</b>
          </h3>
```
FIca mais fácil de ler, o msm se aplica ao doc inteiro